arbor/include/arbor/cable_cell.hpp

```diff
@@ -251,6 +219,20 @@ class cable_cell {
         return parents_;
     }
 
+    // Approximate per-segment mean attenuation b(f) at given frequency f,
+    // ignoring membrane resistance [1/µm].
+    value_type segment_mean_attenuation(value_type frequency, index_type segidx,
+        const cable_cell_parameter_set& global_defaults) const;
+
+    // Estimate of length constant λ(f) = 1/2 · 1/b(f), following
+    // Hines and Carnevale (2001), "NEURON: A Tool for Neuroscientists",
+    // Neuroscientist 7, pp. 123-135.
+    value_type segment_length_constant(value_type frequency, index_type segidx,
```

Comment:
Do we use this function anywhere?

Reply:
This is for use in (external) user code that uses the d_lambda measure for determining the number of CVs to use. The previous method just needed a segment, but now with the inheritable electrical properties, we need to move it out of the segment member functions and make it a cell method.

It's not currently used by any of our examples, but it probably does deserve a unit test.